time timeStamp
float64[] positions